person.rb

```diff
@@ -19,4 +22,9 @@ def can_use_services?
   def of_age?
     @age >= 18
   end
+
+  # Implementing the correct_name method that returns the name
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] You have made a great attempt but the checks are not successful as I could not access your `correct_name` method because it's a private method. Try bringing it above the `private` declaration or use `public :correct_name` to lift the veil.

Reply:
well resolved! ✔️